server/public/model/client4.go

```diff
@@ -6014,8 +6014,13 @@ func (c *Client4) GetOAuthAccessToken(ctx context.Context, data url.Values) (*Ac
 // OutgoingOAuthConnection section
 
 // GetOutgoingOAuthConnections retrieves the outgoing OAuth connections.
-func (c *Client4) GetOutgoingOAuthConnections(ctx context.Context, fromID string, limit int) ([]*OutgoingOAuthConnection, *Response, error) {
-	r, err := c.DoAPIGet(ctx, c.outgoingOAuthConnectionsRoute(), "")
+func (c *Client4) GetOutgoingOAuthConnections(ctx context.Context, fromID string, limit int, audience string) ([]*OutgoingOAuthConnection, *Response, error) {
```

Comment:
Noting this is a breaking change. If it's not avoidable we should make sure to mention it in the release notes.

And if we go ahead with it, we may want to pass an options object (or use `OutgoingOAuthConnectionGetConnectionsFilter` directly if it makes sense to do so) so that we won't need to break it again if we ever add more fields.

Reply:
I'm not sure why I didn't use an object directly, but I'm going with that since that's the option I would prefer as a consumer.